src/configuration/routes/cache.md

```diff
@@ -178,3 +176,5 @@ default value.
 If the response does not have a `Cache-Control` header, the
 `default_ttl` key is used to define the cache duration, in seconds. The
 default value is `0`.
+
+If the application code returns a Cache-Control header, or if your `.platform.app.yaml` file is configured to set a cache lifetime for static assets, then this value is ignored in favor of the application container's headers.
```

Comment:
Per Slack convo and the expires header which _can_ be set in web.locations, I can let this go now...